src/_posts/security/cryptography/2000-01-01-tls.md

```diff
@@ -0,0 +1,24 @@
+---
+title: TLS
+nav: TLS Version
```

Comment:
I don't think it's useful to have a different title for the nav bar and for the page. I think they could match as the length of the title is not that big
```suggestion
title: TLS Version
```